src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -733,20 +761,30 @@ public static View createBuildingView(ColonyView colony, BlockPos id, ByteBuf bu
             try
             {
                 view.deserialize(buf);
-            }
-            catch (IndexOutOfBoundsException ex)
+            } catch (IndexOutOfBoundsException ex)
             {
-                Log.logger.error(
-                        String.format("A AbstractBuilding View (%s) has thrown an exception during deserializing, its state cannot be restored. Report this to the mod author",
-                                oclass.getName()), ex);
+                Log.logger.error(String
+                        .format("A AbstractBuilding View (%s) has thrown an exception during deserializing, its state cannot be restored. Report this to the mod author", oclass
+                                .getName()), ex);
                 view = null;
             }
-        }
-        else
+        } else
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this "else" keyword to a new dedicated line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ARightCurlyBraceDifferentLineAsNextBlockCheck)